healthcheck/ticker.go

```diff
@@ -42,11 +42,14 @@ func (ticker *ticker) start(ctx context.Context, wg *sync.WaitGroup) {
 			}
 		}
 	}()
+
+	// run check as a one of on application start
+	wg.Add(1)
```

Comment:
I know it shouldn't have a huge impact, but I think this should be moved to the go-routine (between line 33 and 34, before the loop), so that we don't block the main thread.
If the use calls Stop(), the main thread will wait in the waitgroup anyway